conf/application.conf

```diff
@@ -43,6 +43,14 @@ play.modules.enabled += "modules.Module"
 
 application.router = prod.Routes
 
+sa.ats.service_unavailable.title="Sorry, the service is unavailable"
+sa.ats.service_unavailable.heading="Sorry, the service is unavailable"
```

Comment:
Is this duplication required? 

We wouldn't change the page title in a config message, I would only have the paragraph editable, the title and header generally remain the same.